src/libmongoc/tests/test-mongoc-rpc.c

```diff
@@ -508,7 +508,7 @@ test_mongoc_rpc_reply_scatter2 (void)
    ASSERT (rpc.reply.documents_len == 16200);
    reader =
       bson_reader_new_from_data (rpc.reply.documents, rpc.reply.documents_len);
-   while ((b = bson_reader_read (reader, &eof))) {
+   while (bson_reader_read (reader, &eof)) {
```

Comment:
Declaration of `const bson_t *b;` on line 484 should be removed as the variable is no longer used.

Reply:
Done.